src/main/resources/config-schema-storage-pravega-kvs.yaml

```diff
@@ -0,0 +1,30 @@
+---
+storage:
+  driver:
+    control:
+      scope: boolean
+      timeoutMillis: long
+    event:
```

Comment:
will we change "event" on smth else?

Reply:
Sure. I guess we can do that already. storage-driver-hashing-key-enabled sounds good enough to me. you?